plc4j/integrations/apache-nifi/README.md

```diff
@@ -85,29 +85,25 @@ Also, it is important to keep in mind the Processor Scheduling Configuration. Us
 
 Table of data mapping between plc data and Avro
 
+
 | PLC type | Avro Type |
-|----------|-----------|
-| PlcBigDecimal | float |
-| PlcBigInteger | long |
-| PlcBitString | string |
+|----------:|-----------|
 | PlcBOOL | boolean |
```

Comment:
should be byte[]?

Reply:
In Avro specification (https://avro.apache.org/docs/1.11.1/specification/#primitive-types) the primitive type is called bytes